### PR TITLE
Updated readme file and ts config for display errors

### PR DESCRIPTION
**Pull request by IsmaelTerreno**
- Updated developer docs with requirements.
- Updated tsconfig.json to reduce more the TS errors when run the od app.

### CODE REVIEW

**Review by pi0neerpat — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -70,3 +77,9 @@ yarn test:e2e
 ```bash
 yarn test
 ```
+
+## Deployments
+
+- http://app.opendollar.com/
+- dev branch http://app.optimism-goerli.opendollar.com/
```
Comment (jahabeebs) [inline README.md:84]: We're not using optimism or optimism/arbitrum goerli anymore

File: yarn.lock [hunk 2]
```diff
@@ -5192,7 +5192,7 @@ __metadata:
     "@ethersproject/experimental": 5.4.0
     "@ethersproject/providers": 5.4.5
     "@gnosis.pm/safe-apps-web3-react": ^0.6.0
-    "@opendollar/sdk": ^1.5.8-rc.5
+    "@opendollar/sdk": ^1.5.8-rc.4
```
Comment (jahabeebs) [inline yarn.lock:5195]: Why are we downgrading the SDK version in the yarn.lock but not the package.json?

**Review by jahabeebs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IsmaelTerreno) [reply README.md:84]: I will take a look on the latest url we use. Thanks @jahabeebs !

File: yarn.lock (hunk 2 above)
Comment (IsmaelTerreno) [reply yarn.lock:5195]: Nice catch, I agree on this one. I think was a local test change that was not supposed to be on this PR. Thanks again @jahabeebs !

File: tsconfig.json [hunk 3]
```diff
@@ -14,6 +14,9 @@
         "resolveJsonModule": true,
         "isolatedModules": true,
         "noEmit": true,
+        "noEmitOnError": true,
```
Comment (pi0neerpat) [inline tsconfig.json:17]: These changes may not be necessary, and we may close this PR, depending on the results of #214

**Pull request closed without merging**